recipes/server.rb

```diff
@@ -28,8 +28,23 @@
 
 package [ 'avahi-daemon', 'avahi-utils' ] if platform_family?('debian')
 
+dpkg_package 'plexmediaserver' do
+  source cached_package_file
+  notifies :restart, 'service[plexmediaserver]'
+  options '--force-confold'
+  action :install
+  version node['plexapp']['version']
+end if platform_family?('debian')
+
 package 'plexmediaserver' do
   source cached_package_file
+  notifies :restart, 'service[plexmediaserver]'
+end unless platform_family?('debian')
```

Comment:
Why have two (very similar) resources when we should be able to make a single one work?

```ruby
package 'plexmediaserver' do
  source cached_package_file
  notifies :restart, 'service[plexmediaserver]'
  options '--force-confold' if platform_family?('debian')
  version node['plexapp']['version'] if platform_family?('debian')
end
```